imagebuildah/stage_executor.go

```diff
@@ -1092,25 +1117,7 @@ func (s *StageExecutor) historyMatches(baseHistory []v1.History, child *parser.N
 		return false
 	}
 	for i := range baseHistory {
-		if baseHistory[i].CreatedBy != history[i].CreatedBy {
-			return false
-		}
-		if baseHistory[i].Comment != history[i].Comment {
-			return false
-		}
-		if baseHistory[i].Author != history[i].Author {
-			return false
-		}
-		if baseHistory[i].EmptyLayer != history[i].EmptyLayer {
-			return false
-		}
-		if baseHistory[i].Created != nil && history[i].Created == nil {
-			return false
-		}
-		if baseHistory[i].Created == nil && history[i].Created != nil {
-			return false
-		}
-		if baseHistory[i].Created != nil && history[i].Created != nil && *baseHistory[i].Created != *history[i].Created {
+		if !historyEntriesEqual(baseHistory[i], history[i]) {
```

Comment:
lint doesn't like something in this file, I'm not sure what though, it was a rather vague error, or perhaps I've just been staring at this screen too long today.

Reply:
It's my eyes most likely, it's not happy with the test file, lint is OK with this.

Reply:
Piping the file through `goimports -d` indicates that it wanted a stylistic change.  And there was a great deal of swearing in my home.